samples/system-test/quickstart.test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const assert = require('assert');
+const path = require('path');
+const test = require('mocha');
+const tools = require('@google-cloud/nodejs-repo-tools');
+const util = require('util');
+const uuid = require('uuid');
+const cwd = path.join(__dirname, '..');
+const cmd = 'node quickstart.js';
+
+const {Storage} = require('@google-cloud/storage');
+
+const storage = new Storage();
+const bucketName = `asset-nodejs-${uuid.v4()}`;
+const bucket = storage.bucket(bucketName);
+
+test.describe('quickstart sample tests', () => {
```

Comment:
You can just leave this as `describe`, and `before`, and `it`

Reply:
Thanks. Not sure why I got lint errors before; now I do see that lint is fine with test removed. Do you want me to delete this fork and create a new fork to commit the change, or just re-use this fork to commit the change?

Reply:
Nah, don't worry about :) Thanks for being patient!

Reply:
Cool. Thanks for the code review!